src/components/GalleryLightBox/GalleryLightBox.tsx

```diff
@@ -81,6 +86,15 @@ export const GalleryLightBox = ({
 
```

Comment:
Changes from this point onwards are largely moving things around as opposed to massive markup overhaul